Source/ProgressionSystemRuntime/Private/Widgets/PSOverlayWidget.cpp

```diff
@@ -108,5 +113,6 @@ void UPSOverlayWidget::TickPlayFadeOverlayAnimation()
 void UPSOverlayWidget::SetOverlayItemsVisibility(ESlateVisibility VisibilitySlate)
 {
 	// Level is unlocked hide the blocking overlay
+	SetVisibility(VisibilitySlate);
 	PSCOverlay->SetVisibility(VisibilitySlate);
```

Comment:
Please remove changing visibility of PSCOverlay `PSCOverlay->SetVisibility(VisibilitySlate);`, as now you hide the widget itself, so no need to hide its overlay component

Reply:
done with: https://github.com/h4rdmol/ProgressionSystem/pull/34/commits/682403d36829f78f030cec4787683d9440153519